frontend/src/components/Router.tsx

```diff
@@ -107,9 +108,10 @@ export const RoutePage = {
   PIPELINES: '/pipelines',
   PIPELINE_DETAILS: `/pipelines/details/:${RouteParams.pipelineId}/version/:${RouteParams.pipelineVersionId}?`,
   PIPELINE_DETAILS_NO_VERSION: `/pipelines/details/:${RouteParams.pipelineId}?`, // pipelineId is optional
-  RECURRING_RUN: `/recurringrun/details/:${RouteParams.runId}`,
   RUNS: '/runs',
   RUN_DETAILS: `/runs/details/:${RouteParams.runId}`,
+  JOBS: '/jobs',
+  JOB_DETAILS: `/jobs/details/:${RouteParams.runId}`,
```

Comment:
Note, so far, we've been calling them `Recurring Runs` consistently in KFP UI. I'm curious why you wanted to change that to jobs?

I understand that there're some confusion between `recurring runs` and `jobs`. However, I like the name `recurring run` more, because `job` is harder to be perceived as sth that runs at a schedule.
We might want to make a naming change uniformly from UI to API, but I'd prefer delaying that in a batch, and `job` is not a good name we'll stick to.

Therefore, I prefer sticking with recurring runs in the UI for now.